exalearn-api/ExaLearn.Bl/EmailService/MessageBuilder.cs

```diff
@@ -0,0 +1,62 @@
+﻿using ExaLearn.Dal.Entities;
+using Shared.Enums;
+using System;
+using System.Net.Mail;
+
+namespace ExaLearn.Bl.EmailService
+{
+    public static class MessageBuilder
+    {
+        public static MailMessage GenerateMessageForUserAsync(
+            User user,
+            LevelType levelType,
+            DateTime expirationDate,
+            bool isAssing
+            )
+        {
+            return CreateEmailMessage(user, levelType, expirationDate, isAssing);
+        }
+
+        public static MailMessage CreateEmailMessage(
+            User user,
+            LevelType levelType,
+            DateTime expirationDate,
+            bool isAssing
+            )
+        {
+            var from = new MailAddress("userexa@mailnesia.com", "supportExaLearn-api");
+            var to = new MailAddress(user.UserName);
+            var fullName = $"{user.FirstName} {user.LastName}";
+            var message = GenerateMessageTemplate(from, to, levelType, expirationDate, fullName, isAssing);
+
+            return message;
+        }
+
+        public static MailMessage GenerateMessageTemplate(
+            MailAddress from,
+            MailAddress to,
+            LevelType levelType,
+            DateTime expirationDate,
+            string fullName,
+            bool isAssing
+            )
+        {
+            var message = new MailMessage(from, to)
+            {
+                Subject = isAssing ? "You are assigned to a test" : "Your a test results",
+                Body = isAssing
```

Comment:
EmailBodyGenerator with 2 method return template and 1 method with switch

Reply:
done